test/NonSilo.Tests/AssemblyLoaderTests.cs

```diff
@@ -5,14 +5,15 @@
 using System.Text;
 using Orleans.Providers;
 using Orleans.Runtime;
+using Orleans.TestingHost.Utils;
 using Xunit;
 
 namespace UnitTests
 {
     public class AssemblyLoaderTests 
     {
         const string ExpectedFileName = "OrleansProviders.dll";
-        private readonly Logger logger = LogManager.GetLogger("AssemblyLoaderTests", LoggerType.Application);
+        private readonly Logger logger = new LoggerWrapper<AssemblyLoaderTests>(TestingUtils.CreateDefaultLoggerFactory($"AssemblyLoaderTests.log"));
```

Comment:
If you are newing up a LoggingFactory, please implement IDisposable and make sure to uninitialize it